Successfully run the power_by_squaring.py. With argument(s) ['2', '10'] got the output of 1024.0.
Elapsed time: 0.06s
Successfully run the power_by_squaring.py. With argument(s) ['123456789', '0'] got the output of 1.0.
Elapsed time: 0.07s
Failed run of the power_by_squaring.py with ['1.001', '1000']. The output should be 2.71692393224, but got 2.716923932235562.
Elapsed time: 0.06s
Failed run of the power_by_squaring.py with ['1.0001', '10000']. The output should be 2.71814592682, but got 2.7181459268244557.
Elapsed time: 0.06s
Failed run of the power_by_squaring.py with ['1.00001', '100000']. The output should be 2.71826823719, but got 2.718268237195739.
Elapsed time: 0.06s
Failed run of the power_by_squaring.py with ['1.000001', '1000000']. The output should be 2.7182804691, but got 2.7182804691276115.
Elapsed time: 0.09s
Failed run of the power_by_squaring.py with ['1.0000001', '10000000']. The output should be 2.71828169413, but got 2.7182816940044554.
Elapsed time: 0.17s
Failed run of the power_by_squaring.py with ['1.00000001', '100000000']. The output should be 2.71828179835, but got 2.718281790326462.
Elapsed time: 2.34s
Failed run of the power_by_squaring.py with ['1.000000001', '1000000000']. The output should be 2.71828205201, but got 2.7182820406313954.
Elapsed time: 29.68s
Failed run of the power_by_squaring.py with ['1.0000000001', '10000000000']. The output should be 2.71828205323, but got 2.7182820448512257.
Elapsed time: 91.95s
